src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -106,13 +106,6 @@ public async Task StopApplicationAsync(CancellationToken cancellationToken = def
         }
     }
 
-    private async Task CreateContainerSingletonsAsync(CancellationToken cancellationToken = default)
-    {
-        // Find containers that consume no Services--their associated Services can be started in Proxyless mode
-        var toCreate = _appResources.Where(r => r.DcpResource is Container && !r.ServicesConsumed.Any());
-        await CreateContainersAsync(toCreate, cancellationToken).ConfigureAwait(false);
-    }
-
```

Comment:
The calling code was commented out, so ideally we'd either comment this, or delete the comment. Ditto below.